printer_mods/al3ph/pl05_mount/README.md

```diff
@@ -0,0 +1,11 @@
+Mount for PL05N2 Sensor.
+
+Designed for this model, but others may fit. https://www.aliexpress.com/item/32805983692.html
+
+Print 1 of each STL.
+

```

Comment:
I'm concerned about what look to be very steep overhangs at the bottom of the magnet mounts on the ducts.
<img width="413" alt="Screen Shot 2021-03-20 at 19 30 30" src="https://user-images.githubusercontent.com/69389593/111888442-e7bc6680-89b2-11eb-8f9a-78badcee859e.png">

Can you modify them so that they're 45 degrees or less?